quinn-udp/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "quinn-udp"
+version = "0.7.0"
+authors = ["Benjamin Saunders <ben.e.saunders@gmail.com>", "Dirkjan Ochtman <dirkjan@ochtman.nl>"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/djc/quinn"
```

Comment:
Should use the quinn-rs URL here.

Reply:
Done. Also fixed repo url in quinn and quinn-proto.